packages/koa-static-cache/CHANGELOG.md

```
@@ -9,7 +9,12 @@
 
 ### ⚠ BREAKING CHANGES
 
-* drop Node.js < 20.19.0 support
+* drop Node.js < 22.18.0 support
+* only support egg@4
+
+part of https://github.com/eggjs/egg/issues/5434
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)